test/lib/ansible_test/_data/sanity/code-smell/runtime-metadata.py

```diff
@@ -184,12 +258,20 @@ def main():
     collection_legacy_file = 'meta/routing.yml'
     collection_runtime_file = 'meta/runtime.yml'
 
+    # This is currently disabled, because if it is enabled this test can start failing
+    # at a random date. For this to be properly activated, we (a) need to be able to return
+    # codes for this test, and (b) make this error optional.
+    check_deprecation_dates = False
```

Comment:
Not sure if I understand the comment - this is to avoid failures for an older collection since the deprecation/tombstone check uses today's date rather than, say, the publish date if there is one?

Reply:
Exactly (except that tombstone dates should always be in the past). Otherwise, suddenly CI for older release branches can start failing because an announced deprecation date ("a major release after xxxx-xx-xx") is suddenly in the past.